theme/mixins/backdrop.less

```diff
@@ -1,11 +1,12 @@
-.backdrop() {
+@backdrop-opacity: 0.7;
+
+.transparrentBackdrop() {
```

Comment:
Curious as to why you are removing the standard backdrop mixin? Sounds like we are looking for either one configurable mixin or two separate mixins — one transparent, one 70% @sk-black.

(Also you have a typo in `transparent` here — double 'r')